lib/ex_vision/style_transfer/style_transfer.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Configuration do
+  @low_resolution {400,300}
+  @high_resolution {640,480}
+  def configuration do
+    %{
+      ExVision.StyleTransfer.Candy  => [model: "candy.onnx", resolution: @high_resolution],
+      ExVision.StyleTransfer.CandyFast  => [model: "candy_fast.onnx", resolution: @low_resolution],
+      ExVision.StyleTransfer.Princess  => [model: "princess.onnx", resolution: @high_resolution],
+      ExVision.StyleTransfer.PrincessFast  => [model: "princess_fast.onnx", resolution: @low_resolution],
+      ExVision.StyleTransfer.Udnie  => [model: "udnie.onnx", resolution: @high_resolution],
+      ExVision.StyleTransfer.UdnieFast  => [model: "udnie_fast.onnx", resolution: @low_resolution],
+      ExVision.StyleTransfer.Mosaic  => [model: "mosaic.onnx", resolution: @high_resolution],
+      ExVision.StyleTransfer.MosaicFast  => [model: "mosaic_fast.onnx", resolution: @low_resolution],
+    }
+  end
+end
+
+for {module, opts} <- Configuration.configuration() do
+  defmodule module do
+    @moduledoc """
+    #{module} is a custom style transfer model optimised for devices with low computational capabilities and CPU inference.
+    """
+    require Logger
+    @type output_t() :: [Nx.Tensor.t()]
```

Comment:
This type is incorrect
```suggestion
    @type output_t() :: Nx.Tensor.t()
```